wormhole-connect/src/consts/wallet.ts

```diff
@@ -0,0 +1,40 @@
+export const SANCTIONED_WALLETS = Object.freeze(
+  new Set([
+    '0x098B716B8Aaf21512996dC57EB0615e2383E2f96',
+    '0xa0e1c89Ef1a489c9C7dE96311eD5Ce5D32c20E4B',
+    '0x3Cffd56B47B7b41c56258D9C7731ABaDc360E073',
+    '0x53b6936513e738f44FB50d2b9476730C0Ab3Bfc1',
+    '0x35fB6f6DB4fb05e6A4cE86f2C93691425626d4b1',
+    '0xF7B31119c2682c88d88D455dBb9d5932c65Cf1bE',
+    '0x3e37627dEAA754090fBFbb8bd226c1CE66D255e9',
+    '0x08723392Ed15743cc38513C4925f5e6be5c17243',
+    '0x7F367cC41522cE07553e823bf3be79A889DEbe1B',
+    '0xd882cfc20f52f2599d84b8e8d58c7fb62cfe344b',
+    '0x5be5543ff73456ab9f2d207887e2af87322c651e',
+    '0x901bb9583b24d97e995513c6778dc6888ab6870e',
+    '0xa7e5d5a720f06526557c513402f2e6b5fa20b008',
+    '0x9f4cda013e354b8fc285bf4b9a60460cee7f7ea9',
+    '0x3cbded43efdaf0fc77b9c55f6fc9988fcc9b757d',
+    '0xfec8a60023265364d066a1212fde3930f6ae8da7',
+    '0x7FF9cFad3877F21d41Da833E2F775dB0569eE3D9',
+    '0xc2a3829F459B3Edd87791c74cD45402BA0a20Be3',
+    '0x3AD9dB589d201A710Ed237c829c7860Ba86510Fc',
+    '0x12D66f87A04A9E220743712cE6d9bB1B5616B8Fc',
+    '0x47CE0C6eD5B0Ce3d3A51fdb1C52DC66a7c3c2936',
+    '0x910Cbd523D972eb0a6f4cAe4618aD62622b39DbF',
+    '0xA160cdAB225685dA1d56aa342Ad8841c3b53f291',
+    '0xD4B88Df4D29F5CedD6857912842cff3b20C8Cfa3',
+    '0xFD8610d20aA15b7B2E3Be39B396a1bC3516c7144',
+    '0x07687e702b410Fa43f4cB4Af7FA097918ffD2730',
+    '0x23773E65ed146A459791799d01336DB287f25334',
+    '0x22aaA7720ddd5388A3c0A3333430953C68f1849b',
+    '0x03893a7c7463AE47D46bc7f091665f1893656003',
+    '0x2717c5e28cf931547B621a5dddb772Ab6A35B701',
+    '0xD21be7248e0197Ee08E0c20D4a96DEBdaC3D20Af',
+    '0x39D908dac893CBCB53Cc86e0ECc369aA4DeF1A29',
+    '0x4f47bc496083c727c5fbe3ce9cdf2b0f6496270c',
+    '0x38735f03b30FbC022DdD06ABED01F0Ca823C6a94',
+    '0x97b1043abd9e6fc31681635166d430a458d14f9c',
+    '0xb6f5ec1a0a9cd1526536d3f0426c429529471f40',
```

Comment:
This seems to be fewer than what I was thinking.  Maybe that's all though

Reply:
This is it for EVM addresses; check out Tony's comment here https://github.com/wormhole-foundation/wormhole-connect/issues/674#issuecomment-1581099300